app/js/components/labOrderEntry/DraftOrder/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import DraftList from './DraftList';
+
+export class LabDraftOrder extends React.Component {
+  render() {
+    const { draftLabOrders, disableCancelButton, disableSaveButton } = this.props;
+    const lengthOfDraftLabOrders = draftLabOrders.length;
```

Comment:
you could change the `lengthOfDraftLabOrders` to `numberOfDraftOrders` so it's more representative

Reply:
cool